src/utils/helper.js

```diff
@@ -1,3 +1,34 @@
-export const selectBoxClicked = () => {
-  console.log('I chose the box not clicked');
+import _ from 'lodash';
+
+export const winPositions = () => {
+  return [
+    [0, 1, 2],
+    [0, 3, 6],
+    [0, 4, 8],
+    [1, 4, 7],
+    [2, 4, 6],
+    [2, 5, 8],
+    [3, 4, 5],
+    [6, 7, 8],
+  ];
+}
+
+export const compareEachWinVariation = (winPosition, playerPositions) => {
+  const bucket = [];
+  if (Array.isArray(playerPositions) && playerPositions.length > 2) {
+    playerPositions.map(position => {
+      _.indexOf(winPosition, position) !== -1 ? bucket.push(true) : null;
```

Comment:
There is a native array method for this. and this seems like you are doing a n2. can you refactor this to be more clear.
and also there seem to be a lot of mutation going on with bucket here